src/main/java/org/jahia/modules/sam/healthcheck/probes/JahiaExceptionsProbe.java

```diff
@@ -0,0 +1,81 @@
+package org.jahia.modules.sam.healthcheck.probes;
+
+import org.jahia.modules.sam.Probe;
+import org.jahia.modules.sam.ProbeSeverity;
+import org.jahia.modules.sam.ProbeStatus;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.text.MessageFormat;
+import java.util.Map;
+
+/**
+ * Probe to be used to get the number of exceptions in a log file.
+ * This probe have a LOW severity level.
+ * The probe returns a yellow level if there is at least one ERROR found in the log file.
+ * By default the log file which is used is located to System.getProperty("jahia.log.dir") + "jahia.log"
+ */
+@Component(immediate = true, service = Probe.class)
+public class JahiaExceptionsProbe implements Probe {
+
+    private static final Logger logger = LoggerFactory.getLogger(JahiaExceptionsProbe.class);
+
+    private String filePath;
+
+    private ProbeSeverity severity = ProbeSeverity.LOW;
+
+    private static final MessageFormat yellowMessage = new MessageFormat("A total of {0} exceptions are present on the platform, exceptions are not expected in a production environment and we recommend reviewing these.");
```

Comment:
Are exceptions the same than errors ? i.e. can there be errors in the log file without an exception ? If that's the case, we should adapt the phrasing. (I'm not against changing the probe to JahiaErrorsProbe and only mention Errors, not exception).
But if all errors are always associated with an exception, then you can disregard this comment.

Reply:
Exceptions are not the same than errors. It can happen that there is error which not show the stacktrace, so it not shows the exception. 
I will change the change to JahiaErrorsProbe

Reply:
sounds good thanks, make sure to remove every mention of exceptions in the probe.